### PR TITLE
Create control plane with cloudformation

### PR DESCRIPTION
**Pull request by nckturner**
- retains old behavior with flag but defaults to new
- copied assets into a new asset dir 1.10.3/2018-07-18
- Fixes https://github.com/weaveworks/eksctl/issues/76

### CODE REVIEW

**Comment by errordeveloper** [comment]
This is great, thanks a lot! I was just thinking of looking into it myself. I will review later on.

**Comment by errordeveloper** [comment]
@nckturner actually, I'm not sure we need the flag, because it's a change in implementation details, and users should rely on such a thing, unless you thinks it fundamentally different?

File: pkg/eks/cfn.go [hunk 1]
```diff
@@ -289,6 +290,81 @@ func (c *ClusterProvider) DeleteStackServiceRole() error {
 	return nil
 }
 
+func (c *ClusterProvider) stackNameDefaultControlPlane() string {
+	return "EKS-" + c.cfg.ClusterName + "-DefaultControlPlane"
```
Comment (errordeveloper) [inline pkg/eks/cfn.go]: just `-ControlPlane`, I think...

**Comment by nckturner** [comment]
@errordeveloper yeah I would have left the flag out had it been my project, but I wasn't sure if backwards compatibility was required, so I threw it in for good measure.  Would you like me to leave it in to be removed in the next release or so, or just remove it now?  I'm not sure why someone *wouldn't* want a cfn resource for their cluster, but perhaps there could be scenarios.  It does (should) make deleting the cluster different, as you should do it through the stack.

**Comment by errordeveloper** [comment]
I am with you. Don't think there is a need to keep clusters creation
backwards compatible, but let's make sure deletions are backwards
compatible. We don't need a flag for deletions, we can just attempt
deleting the cluster if is present, otherwise delete the CFN stack. And we
can break this compatibility in the next release for sure.

**Comment by errordeveloper** [comment]
I think the refactoring PR will take a while. We can land this right after 0.1.0, so hopefully before the end of this week! Do you want to try rebasing?

**Comment by errordeveloper** [comment]
And of course, now we have deletion also.

**Comment by nckturner** [comment]
Ah thanks, I was just working on this last night, but you beat me to it!  Sounds good to me!

**Pull request merged**